### DISCUSSION TITLE
How can I access job input/output from inside a resource?

### DISCUSSION THREAD

**Discussion by pedrovieiramiguez** [concourse/concourse#7966] [Help & Support]
It's a simple use-case, but I'm having issues figuring this out.

What I want to do is:

1. Get a git repo
2. Pass and access information inside the git commit message 
3. Either pass to the resource or access the file <repository>/.git/commit_message
4. Use the information in this file to perform actions in my custom resource

For example, I saw this on StackOverflow

![image](https://user-images.githubusercontent.com/70525853/149556497-a92440ad-1141-4c81-938a-bfa997603066.png)

Leading me to believe that the dockerfile resource can somehow access the repository folder. The problem is that when testing this locally, this folder is not available from my resource, at least not in the default working directory. Is it in some other directory?

I looked at the code in the docker resource and apparently it performs a cat on the passed parameter:
![image](https://user-images.githubusercontent.com/70525853/149557260-b1714419-6c6c-4dd6-99e4-56957139dba8.png)

Problem is, I couldn't replicate this. Probably some steps are done before in the bash script, but I'm terribly inexperienced in bash/shell and therefore having a hard time understanding where I'm going wrong.

I'm currently troubleshooting the issue and I think the problem might have the possible root causes:

1. I'm configuring my resource Dockerfile wrongly for this purpose
2. The repository folder is stored somewhere inside of my resource's container but I haven't found it yet

I'm struggling to find information on this, and maybe the answer is very obvious! Therefore I'm leaving this question here so that maybe before I find a solution someone can point me in the right direction.

Thank you for reading :)

**Comment by timota** [comment]
as i understood correctly - you want to access your get resources inside your task. I would recommend you to read and go through examples in this article to understand how inputs/outputs work in concourse 
https://blog.concourse-ci.org/introduction-to-task-inputs-and-outputs/

**Reply by pedrovieiramiguez** [reply]
I managed to solve this by using fly intercept into the worker and finding where the file I wanted to access was located.

I don't know if this is true for everyone, but for future reference in case someone faces a similar issue, the inputs were mounted in the path

`tmp/build/put/<input-name>`
